ui/components/UserPreferences.js

```diff
@@ -152,7 +152,7 @@ function ThemeToggler({ theme, themeSetter, enqueueSnackbar, classes }) {
         checked={themeToggle}
         onChange={themeToggler}
       />{" "}
-      {themeToggle ? "Light Mode" : "Dark Mode"}
+      Dark Mode
```

Comment:
Can we keep a behavior: 
If it is light theme, then we can show the text: "Light Mode" 
And if it is dark theme, then we can show the text: "Dark Mode" 

What are your views? 🤔

Reply:
If it was a **button** only, then it would be better. But since it is a **toggle switch**, "**Dark Mode**" will be a better approach I guess. Refer to this thread for more insights https://layer5io.slack.com/archives/CFGG6U10E/p1687449363485659

Reply:
Cool, I agree with the vote of majority 👌